libbeat/cmd/instance/beat.go

```diff
@@ -669,8 +672,13 @@ func (b *Beat) configure(settings Settings) error {
 	}
 
 	if maxProcs := b.Config.MaxProcs; maxProcs > 0 {
+		logp.Info("Set max procs limit: %v", maxProcs)
 		runtime.GOMAXPROCS(maxProcs)
 	}
+	if gcPercent := b.Config.GCPercent; gcPercent > 0 {
+		logp.Info("Set gc percentage to: %v", gcPercent)
```

Comment:
Is it worth logging if the value is 100? Being that its the default.

Reply:
We only log if it is configured. Might be helpful to see all message when we have to compare configs with logs output in support cases.

Reply:
For the logs that are only logged once on startup (assuming this is correct) we should rather log too much then too little. Makes debugging easier.